app/models/profile.rb

```diff
@@ -90,6 +90,13 @@ def includes_phone_check?
     proofing_components['address_check'] == 'lexis_nexis_address'
   end
 
+  def strict_ial2_proofed?
```

Comment:
I would like to get `includes_liveness_check?` and `includes_phone_check?` out of the public API for the model eventually. First need to chase down all of the existing uses of `includes_liveness_check?`